### ISSUE TITLE
[MU4 Task][Win,Linux] Shortcuts with special symbols for any language should correspond to English keyboard layout 

### ISSUE THREAD

**Issue by abariska** [musescore/MuseScore#12633]
**Describe the bug**
There're wrong shortcuts:
- `<` - Add crescendo
- `>` - Add decrescendo
- `.` - Add dot to note duration
- `/` - Add grace note
- `;` - move to next beat (chords mode)
- `:` - move to previous beat (chords mode)

Use this shortcut list for more information https://musescore.org/sites/musescore.org/files/MuseScore%20Shortcuts.pdf

**Expected behavior**
Shortcuts which use special symbols work on Windows and Linux with non-English keyboard

**Platform information**
 - OS: Windows, Linux

**Comment by abariska** [comment]
@Tantacrul what do you think about this fix for MU4? We already have this fix implemented for Mac but don't have for windows and Linux. As @Eism said this may call serious regression. So, probably, we should move it to MU4.x

**Comment by Tantacrul** [comment]
Yes. I think we should bump to **4.X** if we're expecting complications.
